edb/schema/pointers.py

```diff
@@ -27,6 +27,7 @@
 from edb.edgeql import ast as qlast
 from edb.edgeql import compiler as qlcompiler
 from edb.edgeql import qltypes
+from edb.server import defines
```

Comment:
Let's move the defines out of `server`.  I don't like the cross-pollination

Reply:
OK, moved to a separate file in `schema`